tox.ini

```diff
@@ -12,9 +12,9 @@ deps =
     beautifulsoup4==4.3.2
     mock==1.0.1
     pytest-cache==1.0
-    pytest==2.7.2
-    pytest-cov==1.8.1
-    pytest-django==2.8.0
+    pytest==2.8.3
+    pytest-cov==2.2.0
+    pytest-django==2.9.1
```

Comment:
Can these also be specified as `>=`... I would prefer that, if possible... What do you think?

Also, comment the reason for the bumps in the commit message.